### DISCUSSION TITLE
[Whisper] File .txt saved without timestamp

### DISCUSSION THREAD

**Discussion by cristianoritta** [Purfview/whisper-standalone-win#170] [Q&A] [answered]
![Captura de tela 2024-01-07 221831](https://github.com/Purfview/whisper-standalone-win/assets/5440016/b113c92f-3880-4b3d-9c6d-b05646c48874)
When I execute whisper.exe to transcribe a audio file, in the  console I see te result with timestamps, but in the file saved, it's not apper.

whisper.exe "<file>" --language pt --model "base" --output_dir="<path>" --output_format=txt 

I try with  --word_timestamps True --verbose True, but not work too.

Can someone help me?

**Comment by Purfview** [comment, marked as answer]
Your selected output format `--output_format=txt` is without timestamps.
All other formats are with timestamps.

NOTE:
Same behavior in `whisper-faster.exe` is with `--output_format=text` format.
